src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -307,21 +315,25 @@ const Embedding = (props) => {
           ? (
             <ClusterPopover
               visible
-              popoverPosition={cellCoordintes}
+              popoverPosition={cellCoordinates}
               onCreate={onCreateCluster}
               onCancel={onCancelCreateCluster}
             />
           ) : (
             (cellInfoVisible && cellInfoTooltip.current) ? (
               <div>
                 <CellInfo
+                  width={CELLINFO_WIDTH}
                   componentType={embeddingType}
-                  coordinates={cellCoordintes}
+                  coordinates={cellCoordinates.current}
                   cellInfo={cellInfoTooltip.current}
+                  numTextRows={numCellInfoTextRows}
```

Comment:
Essentially the same as [this comment](https://github.com/hms-dbmi-cellenics/ui/pull/688/files#r852773243), but try to get rid of this prop and get the height of CellInfo from within CellInfo

Reply:
moved